src/ts/components/account/ConfirmPhrase.tsx

```diff
@@ -120,7 +121,11 @@ class ConfirmPhrase extends React.Component<IConfirmPhraseProps, IConfirmPhraseS
         </Message>
 
         <Section>
-          <StyledButton onClick={this.createAccount} disabled={!this.isPhraseConfirmed()}>
+          <StyledButton
```

Comment:
do you think we should make style components connect to store so that they can apply the color schemes directly ?

Reply:
yeah, make sense. do you know how?

Reply:
we can wrap basic component to a component which can connect to store and get app color setting. So the color theme will be applied in basic component rather than the pages uses it.